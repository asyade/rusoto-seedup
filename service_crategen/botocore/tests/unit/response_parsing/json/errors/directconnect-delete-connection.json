{"__type":"DirectConnectClientException","message":"Connection ID foobar has an invalid format."}
