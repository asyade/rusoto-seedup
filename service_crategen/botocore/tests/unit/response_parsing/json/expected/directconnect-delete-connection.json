{
    "ResponseMetadata": {},
    "Error": {
        "Message": "Connection ID foobar has an invalid format.",
        "Code": "DirectConnectClientException"
    }
}
